File: Heat/Models/Numbering.vb
Imports System.ComponentModel.DataAnnotations

Namespace Models

    Public Class Numbering

        Property ID As Integer
        Property Code As String
        Property Description As String
        Property TempSerialSchema As SerialScheme
        Property FinalSerialSchema As SerialScheme
        Property LastTempSerial As SerialNumber
        Property LastFinalSerial As SerialNumber

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/Numbering.cs
namespace Heat.Models
{

    public class Numbering
	{

		public int ID { get; set; }
		public string Code { get; set; }
		public string Description { get; set; }
		public int TempSerialSchemaID { get; set; }
		public virtual SerialScheme TempSerialSchema { get; set; }
		public int FinalSerialSchemaID { get; set; }
		public virtual SerialScheme FinalSerialSchema { get; set; }
		public SerialNumber LastTempSerial { get; set; }
		public SerialNumber LastFinalSerial { get; set; }

	}
}
